src/apis/message/apis.ts

```diff
@@ -0,0 +1,30 @@
+import type {
+  GetMessageRoomsReq,
+  GetMessageRoomsRes,
+  GetMessageReq,
+  GetMessageRes,
+  CreateMessageReq,
+  CreateMessageRes
+} from './types'
+import { http } from '@utils/http'
+
+export const getMessageRooms = (params: GetMessageRoomsReq) =>
+  http.get<GetMessageRoomsReq, GetMessageRoomsRes>('/msgrooms', params)
+
+export const getMessage = (params: GetMessageReq) =>
+  http.get<GetMessageReq, GetMessageRes>(
+    `/msgrooms/${params.msgRoomId}/msgs`,
+    params
+  )
+
+export const createMessage = (
+  messageRoomId: number,
+  params: CreateMessageReq
+) =>
```

Comment:
```suggestion
export const createMessage = (
  { messageRoomId, ...params }: CreateMessageReq
) =>
```
`CreateMessageReq`에 messageRoomId를 추가로 정의해도 좋을 것 같습니다!

Reply:
request body로 보내는 것과 보내지 않는 것을 분리하고 싶어서 요렇게 했는데 포함하는게 좋을까요 ?_?

Reply:
[반영 완료 !](https://github.com/price-offer/offer-fe/pull/196/commits/c334aa356693e396051ebe90551f92c71661cf30#diff-144ce746130f614a176cc54f430dc8784e94faee4a20d6ed54dca0512180f69fR25-R28)

Reply:
오잉! 반영해주셨군요!